scalr/datadog_checks/scalr/config_models/validators.py

```diff
@@ -0,0 +1,9 @@
+# Here you can include additional config validators or transformers
```

Comment:
Why this file is needed?

Reply:
auto generated. files under config_models are generated from spec.yml